README.md

```diff
@@ -1,12 +1,114 @@
-# Google Proxies for Enterprise Certificates
+# Google Proxies for Enterprise Certificates (Preview)
 
-Google Enterprise Certificate Proxies are part of the [Google Cloud Zero Trust architecture][zerotrust] that enables mutual authentication with [client-side certificates][clientcert]. This repository contains a set of proxies/modules that can be used by clients or toolings to interact with certificates that are stored in [protected key storage systems][keystore].
+Google Enterprise Certificate Proxies (ECP) are part of the [Google Cloud Zero Trust architecture][zerotrust] that enables mutual authentication with [client-side certificates][clientcert]. This repository contains a set of proxies/modules that can be used by clients or toolings to interact with certificates that are stored in [protected key storage systems][keystore].
 
 To interact the client certificates, application code should not need to use most of these proxies within this repository directly. Instead, the application should leverage the clients and toolings provided by Google such as [Cloud SDK](https://cloud.google.com/sdk) to have a more convenient developer experience.
 
-## Installation
+## Compatibility
 
-TBD
+Currently ECP is in Preview stage and all the APIs and configurations are **subject to change**.
+
+Tthe following platforms are supported by ECP:
+
+- MacOS (Keychain, PKCS#11)
```

Comment:
MacOS only supports Keychain by design and Windows only supports Window Certificate Store. To hypothetically support PKCS11 on MacOS would require compiling the Linux-targeted PKCS11 ECP code into a MacOS binary, and installing that on to the device (without gCloud bundling support.) Alternatively, we would need to augment the MacOS ECP to support pkcs11 in addition to keychain. Either way, there will be a lot of work and validation to be done in this area, so it's premature to document this. support.